rng-tools.service

```diff
@@ -0,0 +1,26 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
```

Comment:
I don't know what the convention is, but I would not expect this file to be considered part of systemd. If another package adds a unit file, it's part of that package.

Reply:
I just copied over a service file to work from. I can update that.

Reply:
changed to provided by pi-gen